src/Button.php

```diff
@@ -421,7 +470,7 @@ public function render(): string
 
         unset($attributes['class'], $attributes['id']);
 
-        Html::addCssClass($attributes, [self::NAME, $this->buttonVariant->value, $classes]);
+        Html::addCssClass($attributes, [self::NAME, $this->buttonVariant?->value, $classes]);
```

Comment:
Does it make sense to set an empty CSS class in case `buttonVariant` is `null`?

Reply:
There is no problem passing `null`, `addCssClass()` filters out nulls, and when the null variant is used, the base class is simply used. https://getbootstrap.com/docs/5.3/components/buttons/#base-class